quinn-h3/src/client.rs

```diff
@@ -540,6 +544,33 @@ where
     }
 }
 
+#[project]
+impl<B> SendRequest<B, B::Data>
+where
+    B: HttpBody + 'static,
+    B::Error: Into<Box<dyn StdError + Send + Sync>> + Send + Sync,
+{
+    fn take_request(&mut self) -> Result<(Method, Uri, HeaderMap, B), Error> {
```

Comment:
Maybe just name this `take()`? The `_request` suffix feels stuttery on a type that's called `SendRequest`.

Reply:
Oh yeah! I use to rant on this in my dayjob review!